example/.gitignore

```diff
@@ -65,5 +65,5 @@ yarn-error.log
 
 # Env
 .env.development
-/ios/JellyfishExample.xcworkspace/contents.xcworkspacedata
+/ios/FishjamExample.xcworkspace/contents.xcworkspacedata
 /ios/jf/
```

Comment:
Not sure where this line came from, but I think it could be either removed or updated